src/components/Main/Main.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect, useContext } from "react";
+import "./Main.css";
+import homepage from "../../assets/homepage.jpg";
+
+function Main() {
+  const [greeting, setGreeting] = useState("");
+
+  useEffect(() => {
+    const getGreeting = () => {
+      const currentHour = new Date().getHours();
+
+      if (currentHour < 12) {
+        return "good morning";
+      } else if (currentHour < 18) {
+        return "good afternoon";
+      } else {
+        return "good evening";
+      }
+    };
+
+    setGreeting(getGreeting());
+  }, []);
+  return (
+    <main>
+      <div className="main__title">Hello, {greeting} !</div>
```

Comment:
```jsx

      <div className="main__title">Hello, {greeting} !</div>

```

If it's a text, then it should be at least a `p` tag, according to the semantics